### ISSUE TITLE
Not obvious how to configure Docker Daemon with TLS

### ISSUE THREAD

**Issue by joaofnfernandes** [docker/docker.github.io#5167]
We have articles explaining how to set up Docker with mutual TLS, but generally those articles are hard to follow.
I'm sure we can simplify those articles a bit.

https://docs.docker.com/engine/security/https/

**Comment by L-Hudson** [comment]
Closing this ticket due to its age, and the impending refactor. If you think this is in error, feel free to reopen. Thanks!
